[pt] Creating the issues.md in portuguese (#7449)
open-telemetry/opentelemetry.io: content/pt/docs/contributing/issues.md

---
title: Issues
description:
  Como corrigir um problema existente ou relatar um bug, risco de segurança ou possível melhoria.
weight: 10
_issues: https://github.com/open-telemetry/opentelemetry.io/issues
_issue: https://github.com/open-telemetry/opentelemetry.io/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A
cSpell:ignore: prepopulated

> ---

Author: Vou recuperar o hash do commit e adiciono aqui Mary, obrigado.

> ---

Author: Feito ✅ 

Gracias pelo artigo no blog @vitorvasc

## Corrigindo um problema existente

Uma das melhores formas de contribuir para a melhoria da documentação do OTel é corrigir um problema já existente.

1. Navegue pela lista de [issues]({{% param _issues %}}).
2. Selecione uma issue com a qual você gostaria de contribuir, de preferência uma que possa ser resolvida em pouco tempo. <a name="first-issue"></a>
   {{% alert title="É sua primeira contribuição?" color="primary alert--first-timer" %}}

   Procure por issues com os seguintes rótulos:
   - [Good first issue](<{{% param _issue %}}%22good+first+issue%22>)
   - [Help wanted](<{{% param _issue %}}%3A%22help+wanted%22>)

   > **NOTA**: nós **_não atribuímos issues_** a quem ainda não contribuiu com a organização [OpenTelemetry][org], exceto em casos de mentoria ou processos formais de onboarding.
   >
   > [org]: https://github.com/open-telemetry

   {{% /alert %}}

> 3. Leia os comentários existentes na issue, se houver.

Reviewer: ```suggestion
3. Leia os comentários existentes na _issue_, se houver.
```

> 4. Pergunte aos mantenedores se a issue ainda é relevante e esclareça eventuais dúvidas comentando na própria issue.

Reviewer: ```suggestion
4. Pergunte aos mantenedores se a _issue_ ainda é relevante e esclareça eventuais dúvidas comentando na própria _issue_.
```

> 5. Declare sua intenção de trabalhar na issue adicionando um comentário.

Reviewer: ```suggestion
5. Declare sua intenção de trabalhar na _issue_ na forma um comentário na própria _issue_.
```

6. Trabalhe na correção do problema. Caso enfrente dificuldades, avise os mantenedores.
7. Quando estiver pronto, [submeta seu trabalho via pull request](../pull-requests) (PR).

## Reportando um problema

> Se você encontrar um erro ou quiser sugerir melhorias em conteúdos existentes, abra uma nova issue.

Reviewer: ```suggestion
Se você encontrar um erro ou quiser sugerir melhorias em conteúdos existentes, abra uma nova _issue_.
```

> 1. Clique no link **Create documentation issue** disponível em qualquer documento. Isso redirecionará você para uma página no GitHub com um modelo de issue pré-preenchido.

Reviewer: ```suggestion
1. Clique no link **Create documentation issue (Criar _issue_ de documentação)** disponível em qualquer documento. Isso redirecionará você para uma página no GitHub com um modelo de _issue_ pré-preenchido.
```

2. Descreva o problema ou sugestão de melhoria. Forneça o máximo de detalhes possível.

> 3. Clique em **Create**.

Reviewer: ```suggestion
3. Clique em **Create (Criar)**.
```

> Após enviar a issue, acompanhe as atualizações ou ative as notificações do GitHub. Pode levar alguns dias até que mantenedores e aprovadores respondam. Revisores e membros da comunidade podem fazer perguntas antes de agir sobre sua issue.

Reviewer: ```suggestion
Após enviar a _issue_, acompanhe as atualizações ou ative as notificações do GitHub. Pode levar alguns dias até que mantenedores e aprovadores respondam. Revisores e membros da comunidade podem fazer perguntas antes de agir sobre sua _issue_.
```

## Sugerindo novos conteúdos ou funcionalidades

> Se você tem uma ideia para um novo conteúdo ou funcionalidade, mas não sabe exatamente onde ela se encaixa, ainda assim pode abrir uma issue. Também é possível relatar bugs ou vulnerabilidades de segurança.

Reviewer: ```suggestion
Se você tem uma ideia para um novo conteúdo ou funcionalidade, mas não sabe exatamente onde ela se encaixa, ainda assim pode abrir uma _issue_. Também é possível relatar bugs ou vulnerabilidades de segurança.
```

1. Acesse a aba de [Issues no GitHub](https://github.com/open-telemetry/opentelemetry.io/issues/new/).

> 

Reviewer: ```suggestion
```

> 2. Clique em **New issue**.

Reviewer: ```suggestion
2. Clique em **New issue (Nova _issue_)**. 
```

> 

Reviewer: ```suggestion
```

> 3. Escolha o tipo de issue mais apropriado para sua sugestão ou dúvida.

Reviewer: ```suggestion
3. Escolha o tipo de _issue_ mais apropriado para sua sugestão ou dúvida.
```

> 

Reviewer: ```suggestion
```

4. Preencha o template.

> 

Reviewer: ```suggestion
```

> 5. Envie a issue.

Reviewer: ```suggestion
5. Envie a _issue_.
```

### Como escrever boas issues

> Ao criar uma issue, considere as seguintes boas práticas:

Reviewer: ```suggestion
Ao criar uma _issue_, considere as seguintes boas práticas:
```

- Forneça uma descrição clara do problema. Explique exatamente o que está incorreto, desatualizado, ausente ou pode ser melhorado.
- Descreva o impacto específico do problema para os usuários.

> - Mantenha o escopo da issue razoável. Problemas muito amplos devem ser divididos em partes menores. Por exemplo, "Corrigir a documentação de segurança" é vago demais; "Adicionar detalhes ao tópico 'Restringindo o acesso à rede'" é específico e executável.

Reviewer: ```suggestion
- Limite o escopo da _issue_ para uma quantia razoável. Problemas com escopo muito amplo devem ser divididos em _issues_ menores. Por exemplo, "Corrigir a documentação de segurança" é vago demais; "Adicionar detalhes ao tópico 'Restringindo o acesso à rede'" é específico o suficiente para ser acionável.
```

> - Pesquise entre as issues existentes para evitar duplicações.

Reviewer: ```suggestion
- Pesquise entre as _issues_ existentes para ver se não há outra relacionada ou semelhante a sua nova _issue_.
```

> - Se a nova issue estiver relacionada a outra issue ou pull request, faça referência usando a URL completa ou o número precedido por `#`, por exemplo: `Introduzido por #987654`.

Reviewer: ```suggestion
- Se a nova _issue_ estiver relacionada a outra _issue_ ou _pull request_, faça referência usando a URL completa ou o número precedido por `#`, por exemplo: `Introduzido por #987654`.
```
